.github/workflows/pypi.yml

```diff
@@ -21,15 +21,25 @@ jobs:
   make_sdist:
     name: Make SDist
     runs-on: ubuntu-latest
+    permissions:
+      # write id-token and attestations are required to attest build provenance
+      id-token: write
+      attestations: write
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
           submodules: true
+          persist-credentials: false
 
       - name: Build SDist
         run: pipx run build --sdist
 
+      - name: Attest GitHub build provenance
```

Comment:
Over at the pymc repo you had unified the sdist and bdist_wheel steps into a single check from a different tool. Why not do that here too?

Reply:
Compilation.

From the `build-and-inspect-python-package` readme:
![image](https://github.com/user-attachments/assets/7398e6aa-9796-4bc6-a059-21c8454e9d90)